CONTRIBUTING.md

```diff
@@ -30,7 +30,9 @@ The easiest way to get involved with the active development of AutoEmulate is to
 
 ## How to Contribute
 
-We welcome contributions of all kinds, be it code, documentation, or community engagement. We encourage you to read through the following sections to learn more about how you can contribute to
+We welcome contributions of all kinds, be it code, documentation, or community engagement. We encourage you to read through the following sections to learn more about how you can contribute to the package.
+
+We are always interested in adding more sampling/experimental design strategies (see https://github.com/alan-turing-institute/autoemulate/issues/6) and more simulation data packages (see https://github.com/alan-turing-institute/autoemulate/issues/4).
```

Comment:
I think we can take out the first bit about design strategies. Most sampling/experimental design strategies are implemented already in some software, so we just need to chose a few more.

But it'd be great to have more simulations, or simulation input/output datasets from any field, let's maybe focus this on the second part.

Reply:
See https://github.com/alan-turing-institute/autoemulate/pull/156/commits/ebaa98a93fb6315fe8a87176de024c9b764bd60b